lib/routes/universities/wzu/news.js

```diff
@@ -1,24 +1,78 @@
 const got = require('@/utils/got');
 const cheerio = require('cheerio');
+const { URL } = require('url');
 const { parseDate } = require('@/utils/parse-date');
 
+/* 新闻列表
+温大新闻 http://www.wzu.edu.cn/index/wdxw.htm
+媒体温大 http://www.wzu.edu.cn/index/mtwd.htm
+学术温大 http://www.wzu.edu.cn/index/xswd.htm
+通知公告 http://www.wzu.edu.cn/index/tzgg.htm
+招标信息 http://www.wzu.edu.cn/index/zbxx.htm
+学术公告 http://www.wzu.edu.cn/index/xsgg.htm
+*/
+
+const baseUrl = 'http://www.wzu.edu.cn/index/';
+const newsList = ['温大新闻', '媒体温大', '学术温大', '通知公告', '招标信息', '学术公告'];
+const urlList = ['wdxw.htm', 'mtwd.htm', 'xswd.htm', 'tzgg.htm', 'zbxx.htm', 'xsgg.htm'];
+
+/**
+ * @description: 抓取文章内容，本函数参考并修改自 whu/news.js，感谢原作者。
+ * @param {*} link
+ * @return {*}
+ */
+async function load(link) {
+    let description = '';
+    let response;
+    try {
+        // 异步请求文章
+        response = await got.get(link);
+    } catch (err) {
+        // 如果网络问题，直接出错
+        if (err.name && (err.name === 'HTTPError' || err.name === 'RequestError')) {
+            description = 'Page 404 Please Check!';
+        }
+        return description;
+    }
```

Comment:
错误正确抛出，不要内部处理，应该让客户端无法抓取而不是产生垃圾 item
```suggestion
    response = await got.get(link);
```